SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```
@@ -24,20 +24,21 @@ Source0:        %{name}-%{version}.tar.gz
 #           -cf %%{name}-%%{version}-vendor.tar.gz vendor
 #
 Source1:        %{name}-%{version}-vendor.tar.gz
-
+# If upstream ever upgrades client_goland to 1.11.1, we can get rid of this patch.
+Patch0:         CVE-2022-21698.patch
 BuildRequires:  golang >= 1.13
 
 %description
 This is an ingress controller that can be run on Azure Kubernetes Service (AKS) to allow an Azure Application Gateway 
 to act as the ingress for an AKS cluster. 
 
 %prep
-%autosetup
-
-%build
+%setup
```

Just an FYI, we can keep `%autosetup` by adding the `-N` flag:
```suggestion
%autosetup -N
```

---

Thanks. I missed that in the docs.